packages/jaeger-ui-components/src/demo/trace-generators.js

```diff
@@ -149,7 +151,7 @@ export default chance.mixin({
       operationName: chance.pickone(operations),
       references: [],
       startTime,
-      duration: chance.integer({ min: 1, max: traceEndTime - startTime }),
+      duration: chance.integer({ min: 1, max: maxDuration === 0 ? 2 : maxDuration }),
```

Comment:
Shouldn't we check for `maxDuration < 1`? Does it break only when `maxDuration === 0` or when it's lower (or equal?) the minimum (1)?

Reply:
You are right! we should check for maxDuration <= 1